CONTRIBUTING.md

```diff
@@ -4,7 +4,6 @@
 
 <p align="center">
 <a href="https://github.com/mfem/mfem/blob/master/LICENSE"><img alt="License" src="https://img.shields.io/badge/License-BSD-brightgreen.svg"></a>
-<a href="https://travis-ci.org/mfem/mfem"><img alt="Build Status" src="https://travis-ci.org/mfem/mfem.svg?branch=master"></a>
```

Comment:
Is there an equivalent GH actions badge?

Reply:
@tzanio I had missed that, so I created https://github.com/mfem/mfem/pull/2414.